mentoring/frontend/views.py

```diff
@@ -7,3 +9,24 @@
 @user_passes_test(lambda user: user.is_authenticated)
 def root(request):
     return render(request, 'frontend/root.html', {})
+
+
+# Pass some settings off to the browser for use in the UI
+@user_passes_test(lambda user: user.is_authenticated)
+def settings(request):
+    user = request.user
+    settings = {
+        "csrftoken": "uhh",  # TODO: get from cookie - https://docs.djangoproject.com/en/3.1/ref/csrf/
```

Comment:
I think you want to call ``django.middleware.csrf.get_token()`` to extract the token from the cookie:

https://github.com/django/django/blob/d828beb68f6b925217242e8f4faac2216f7c9c57/django/middleware/csrf.py#L74-L90

Reply:
hmm I just realized I was two clicks away from trying this as a fix. Let me see if it works...

Reply:
yes, this makes the form work again:

```py
from django.middleware.csrf import get_token

...

    settings = {
        "csrftoken": get_token(request),
```